test/performance/alignment/local_affine_alignment_benchmark.cpp

```diff
@@ -0,0 +1,277 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+#include <benchmark/benchmark.h>
+
+#include <iostream>
+#include <memory>
+#include <random>
+#include <utility>
+#include <vector>
+
+#include <seqan3/alignment/pairwise/align_pairwise.hpp>
+#include <seqan3/alphabet/aminoacid/aa20.hpp>
+#include <seqan3/alphabet/nucleotide/dna4.hpp>
+
+#if __has_include(<seqan/align.h>)
+    #define SEQAN3_HAS_SEQAN2 1
+#endif
+
+#ifdef SEQAN3_HAS_SEQAN2
+    #include <seqan/align.h>
+    #include <seqan/basic.h>
+    #include <seqan/sequence.h>
+#endif
+
+using namespace seqan3;
+
+template <typename alphabet_t>
+auto generate_sequence_seqan3(size_t const len = 500,
```

Comment:
This should be refactored and DRYied out. There is an open PR #755 which does extracts this part already. We are currently working on getting this in and then it would be good to use the global sequence generation for the local and the global alignment.